internal/adapters/sqlite/integrity.go

```
@@ -340,6 +341,27 @@ func checkIntegrityFamilyTx(ctx context.Context, tx *sql.Tx, family application.
 		if err := queryIDs(`SELECT l.repository FROM repository_lifecycles l WHERE l.current_snapshot_id<>'' AND NOT EXISTS(SELECT 1 FROM repository_readiness_snapshots s WHERE s.snapshot_id=l.current_snapshot_id AND s.incarnation_id=l.incarnation_id) ORDER BY l.repository LIMIT ?`, "readiness_pointer_violation", "repository", application.ScopeRepository); err != nil {
 			return result, nil, err
 		}
+		if err := queryIDs(`SELECT c.onboarding_id FROM repository_onboarding_step_claims c JOIN repository_onboarding_steps s ON s.onboarding_id=c.onboarding_id AND s.step_name=c.step_name JOIN repository_onboardings o ON o.onboarding_id=c.onboarding_id WHERE c.intent_digest<>s.intent_digest OR c.attempt_number>s.attempt_number OR (c.status='active' AND (c.attempt_number<>s.attempt_number OR s.status<>'intended' OR o.step_index+1<>s.step_order)) OR (c.status='settled' AND c.attempt_number=s.attempt_number AND s.status<>'observed') OR (c.status='superseded' AND NOT EXISTS(SELECT 1 FROM repository_onboarding_step_claims n WHERE n.onboarding_id=c.onboarding_id AND n.step_name=c.step_name AND n.attempt_number=c.attempt_number AND n.claim_version=c.claim_version+1)) OR c.claim_version<>(SELECT COUNT(*) FROM repository_onboarding_step_claims p WHERE p.onboarding_id=c.onboarding_id AND p.step_name=c.step_name AND p.attempt_number=c.attempt_number AND p.claim_version<=c.claim_version) ORDER BY c.onboarding_id,c.step_name,c.attempt_number,c.claim_version LIMIT ?`, "onboarding_claim_binding_violation", "onboarding_claim", application.ScopeOnboarding); err != nil {
+			return result, nil, err
+		}
+		claimRows, err := tx.QueryContext(ctx, `SELECT onboarding_id,step_name,attempt_number,intent_digest,supervisor_id,execution_nonce,claim_version,claim_digest FROM repository_onboarding_step_claims ORDER BY onboarding_id,step_name,attempt_number,claim_version LIMIT ?`, application.IntegrityMaximumLimit+1)
```

golangci-lint (2.13.2)

[error] rows.Err must be checked

(rowserrcheck)